components/contribution-flow/ContributionBlocker.js

```diff
@@ -71,6 +72,8 @@ export const getContributionBlocker = (loggedInUser, account, tier, shouldHaveTi
     return { reason: CONTRIBUTION_BLOCKER.TIER_EXPIRED, type: 'warning', showOtherWaysToContribute: true };
   } else if (account.settings.disableCustomContributions && !tier) {
     return { reason: CONTRIBUTION_BLOCKER.NO_CUSTOM_CONTRIBUTION, type: 'warning', showOtherWaysToContribute: true };
+  } else if (account.settings.disableCryptoContributions && !tier) {
```

Comment:
Doesn't take into account that the default is true.

Reply:
This condition should not be there actually; I wrongly put it there. The contribution should not be blocked for not having cyrpto contributions enabled. Corrected it.